en/editor/console/index.md

```diff
@@ -0,0 +1,28 @@
+# Console
+
+![main](index/main.jpg)
+
+**Console** display errors, warnings, or other log information generated by the editor and engine. Information of different important levels is displayed in different colors.
+
+## Abourt log
+The log system is based on `js` **console** model. When the editor is running, will captures the necessary logs printed out of the **console** in each process and distinguishes them by color.
+
+  - **Log** (__console.log__): <font color=#b6b6b6>This is gray text</font>, editor and developers think the content of need printing to **console**.
+  - **Warn** (__console.warn__): <font color=#ebbe09>This is yellow text</font>, when the program is running, it encounter abnormal situation that does not affect  the result.
+  - **Error** (__console.error__): <font color=#dd3c43>This is red text</font>, encountered in run abnormal situation that will to affect the result, a problem that must be resolved, or an uncaught error message.
+
+## Panel operation
+
+![main](index/tools.jpg)
+
+All the **console**'s functionality is on a toolbar at the top, the functionaliys from left to right are:
+
+*  Clear all logs in the current **console**
+*  Open log file
+*  Enter text to filter the displayed log
+*  Whether regular matching is used in text filtering
+*  Filter the displayed log types
+
+## Other settings
+
+Other settings for the **console** are in preferences, please refer to the extended settings in [preferences](../preferences/index.md).
```

Comment:
Other settings for the **console** are in preferences, please refer to the extended settings in [preferences](../preferences/index.md).
--> Other settings for the **console** are available in __Preferences__. Read about advanced settings in the [Preferences](../preferences/index.md) documentation.